hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsAsyncQueueService.kt

```diff
@@ -30,8 +31,8 @@ open class HmppsAsyncQueueService(
     private val gson = GsonBuilder().setObjectToNumberStrategy(ToNumberPolicy.LONG_OR_DOUBLE).create()
   }
 
-  private val hmppsAsyncTopics: List<HmppsAsyncTopic> = hmppsAsyncTopicFactory.createHmppsAsyncTopics(hmppsSqsProperties)
-  private val hmppsAsyncQueues: List<HmppsAsyncQueue> = hmppsAsyncQueueFactory.createHmppsAsyncQueues(hmppsSqsProperties, hmppsAsyncTopics)
+  private val hmppsAsyncTopics: List<HmppsAsyncTopic> = runBlocking { hmppsAsyncTopicFactory.createHmppsAsyncTopics(hmppsSqsProperties) }
+  private val hmppsAsyncQueues: List<HmppsAsyncQueue> = runBlocking { hmppsAsyncQueueFactory.createHmppsAsyncQueues(hmppsSqsProperties, hmppsAsyncTopics) }
```

Comment:
I think it's fair to block here because this happens during application startup and we need to know it's finished before the bean is made available.